integration_test/cleanup/test-cleanup.py

```diff
@@ -0,0 +1,77 @@
+import boto3
+import os
+from orca_shared.database import shared_db
+from sqlalchemy import text
+from aws_lambda_powertools import Logger
+from botocore.exceptions import ClientError
+
+
+LOGGER = Logger()
+
+# os.env keys
+recovery_bucket_name = os.environ["orca_RECOVERY_BUCKET_NAME"]
+source_bucket_name = os.environ["SOURCE_BUCKET_NAME"]
+db_connect_info_secret_arn = os.environ["DB_CONNECT_INFO_SECRET_ARN"]
+
+# grab the objects to be deleted
+s3 = boto3.client('s3')
+result = s3.list_objects_v2(
+    Bucket=source_bucket_name,
+    Prefix='test'
+)
+filenames = []
+for item in result['Contents']:
+    files = item['Key']
+    filenames.append(files)
+LOGGER.info(filenames)
+
+# Delete objects from source and recovery buckets
+for object in filenames:
+    response_source = s3.delete_object(
+        Bucket=source_bucket_name,
+        Key=object,
+    )
+    response_destination = s3.delete_object(
+        Bucket=recovery_bucket_name,
+        Key=object,
+    )
+
+# verify objects are deleted from buckets
+for object in filenames:
+    try:
+        head_object_output = s3.head_object(
+        Bucket=source_bucket_name,Key=object
+        )
```

Comment:
I like the logging here.

Reply:
There doesn't seem to be a failure state... As-in, if the expected ClientError is not raised, the user is not notified and no error is raised.

Reply:
added failure state

Reply:
Close. I'd recommend removing the `if head_recovery_object_output["ResponseMetadata"]["HTTPStatusCode"] == 200:` check. It adds a side-path that we don't expect, and don't have any code to handle.
```
for object in recovery_filenames:
    try:
        head_recovery_object_output = s3.head_object(
        Bucket=recovery_bucket_name,Key=object
        )
        raise Exception(f"{object} still exists in {recovery_bucket_name}")    
    except ClientError as error:
        if error.response['Error']['Code'] == '404':
            LOGGER.info(f"{object} deleted from {recovery_bucket_name}. Test passed!")
        else:
            raise error
```